### ISSUE TITLE
Homepage: Terminology and order of the "user" buttons is inconsistent with the menu

### ISSUE THREAD

**Issue by Arika-V** [OPENDAP/Website#13]
Description: 
"For users" does not match the terminology used among the other buttons and in the menu; instead it should probably be "for data users" (?? "for data accessors" doesn't sound right either...)
"For data providers" and "For data managers" is listed in a different order than in the header menu

https://newopendap.flywheelsites.com/

![Image](https://github.com/OPENDAP/Website/assets/170548569/ffec5b25-cc56-444b-9434-954f6ddaba60)

**Comment by Mikejmnez** [comment]
I changed the order to be consistent with the menu bar options under "Use Cases". I also changed "For Users" to "For Data Access".

**Comment by Mikejmnez** [comment]
@jgallagher59701 also this

**Comment by jgallagher59701** [comment]
This looks done, but it's a little hard to follow what was changed and how. Bottom line, though, it looks good and I think we can say it's done. Thanks.

**Comment by Mikejmnez** [comment]
Yeah, I agree. It is not possible to write a review, or feedback on the code / approach to solve the problem. We can just verify with a `Yes` or `No`, that the problem is still there. But at least we can track what the problems were, how many have been addressed, and the overall progress.
